### PR TITLE
Remove tty-prompt from project

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-<TODO>

### What?

- Removes tty-prompt gem
- Remove references to tty-prompt

This 1 gem brings in 8 gems and adds very little value - it was giving us a fzf interface for choosing from 3 options when `gets` and a message saying which options are possible works just as well.

I've ran all three variations of this and it works as we expect.

### Why?

I generally prefer to maintain as few deps as possible to make our lives
easier/give us fewer things to worry about from a security point of view

### CODE REVIEW

**Comment by willfish** [comment]
```shell
rake auth:create_client_application

What's the application's name?
qux

Which application owner should your tokens be associated with? ["geoamey", "serco", "none"]
geoamey

Created OAuth2 client with (name: qux)
client_id: pIg9LJZqUWpyVyxchaSBBmBY0xuggCkI0-IMeM33ZFM
client_secret: 9SXJSFBGMSsqY2DI_-5O8w_0HOIWaIhvP0SDUOeDeWM
supplier: geoamey
```

**Comment by willfish** [comment]
```
What's the application's name?
wibble
Which application owner should your tokens be associated with? ["geoamey", "serco", "none"]
none
Created OAuth2 client with (name: piss)
client_id: lp9o8vSQBi5HDFpCfIrGrxhuPwo4C1VFrKizl7BLXfM
client_secret: aa0sbwG0RIlednVafhFvsaKeCIkD5_EAW35PbUGemKM
supplier: none
supplier_id:
```

File: lib/tasks/create_application.rake [hunk 1]
```diff
@@ -1,40 +1,30 @@
...
 
     suppliers = Supplier.all.each_with_object({}) { |supplier, accumulator|
-      accumulator[supplier.name] = supplier.id
-    }.merge(NO_OWNER => nil)
-    owner_id = prompt.select('Is this application owned by a supplier?', suppliers)
-    if owner_id != NO_OWNER
-      owner = Supplier.find(owner_id)
-      application.owner = owner
-    end
+      accumulator[supplier.name.downcase] = supplier.id
```
Comment (willfish) [inline lib/tasks/create_application.rake:14]: Downcasing to make it easier to type in the CLI

Comment (martyn-w) [reply lib/tasks/create_application.rake:14]: no need for downcase: you can do `supplier.key` instead, e.g. 
```
suppliers = Supplier.all.map{|s| [s.key,  s.id]}.to_h.merge('none' => nil)
```

**Pull request merged**